include/lsst/daf/base/DateTime.h

```diff
@@ -63,15 +63,15 @@ class DateTime {
     explicit DateTime(double date, DateSystem system = MJD, Timescale scale = TAI);
     DateTime(int year, int month, int day, int hr, int min, int sec,
              Timescale scale = TAI);
-    explicit DateTime(std::string const& iso8601);
+    explicit DateTime(std::string const& iso8601, Timescale scale);
```

Comment:
@timj While the document mentions TAI as a possible timescale, it's a little wishy-washy about how it should be represented.  Note that 4.2.1 mentions non-UTC timescales but no particular representation, and none of the subsequent subsections that do give representations seem appropriate for a TAI time that is not a local time nor a "UTC of day".